### ISSUE TITLE
Random icons in friendly nameplates while they are disabled.

### ISSUE THREAD

**Issue by Dzinsham** [jordonwow/bigdebuffs#203]
Random CC icons appearing on friendly nameplates, example below. I see it all the time in Oribos for example. None of these players are being controlled or have any kind of the buffs  that's showing on their nameplates. Reloading helps as well as turning on and off ANY bigdebuffs setting - icons just disappears. Friendly nameplates are disabled in BigDebuffs settings.
https://i.imgur.com/1KcSOsh.png

**Comment by ujellyx** [comment]
Do you have ThreatPlates by any chance? There seems to be an incompatibility with it, that unfortunately hasn't been fixed for like 3 months now... :(

**Comment by Dzinsham** [comment]
> Do you have ThreatPlates by any chance? There seems to be an incompatibility with it, that unfortunately hasn't been fixed for like 3 months now... :(

Yes i do. But i asked my friend who's using the same combo (tptp+bd) and he said he has never experienced this bug :/

**Comment by ujellyx** [comment]
> > Do you have ThreatPlates by any chance? There seems to be an incompatibility with it, that unfortunately hasn't been fixed for like 3 months now... :(
> 
> Yes i do. But i asked my friend who's using the same combo (tptp+bd) and he said he has never experienced this bug :/

According the the screenshot you're using "Headline view" I assume. For whatever reason this is the same as having the regular nameplates enabled for BigDebuffs. That's why you're seeing this. Your friend probably has friendly nameplates disabled completely.

**Comment by Dzinsham** [comment]
> > > Do you have ThreatPlates by any chance? There seems to be an incompatibility with it, that unfortunately hasn't been fixed for like 3 months now... :(
> > 
> > 
> > Yes i do. But i asked my friend who's using the same combo (tptp+bd) and he said he has never experienced this bug :/
> 
> According the the screenshot you're using "Headline view" I assume. For whatever reason this is the same as having the regular nameplates enabled for BigDebuffs. That's why you're seeing this. Your friend probably has friendly nameplates disabled completely.
> 
> Given the fact that the author of the addon made one update just including work of someone else after 3 months of nothing is no good omen though :/ I doubt that this'll get fixed soon.

No, he uses the headline view too, he's playing a holy pala in arenas, that helps him see his teammates and distinguish them from enemies since nameplates are different. But yea, that's a strange thing going on.
